resources/lib/Navigation.py

```diff
@@ -608,6 +612,25 @@ def remove_from_library(self, video_id, season=None, episode=None):
         self.kodi_helper.dialogs.show_no_metadata_notify()
         return False
 
+    @log
+    def export_new_episodes(self):
+        no_errors = True
+        for title, meta in self.library.list_exported_shows().iteritems():
+            try:
+                self.log('Exporting new episodes of {} (id={})'
+                         .format(title, meta['netflix_id']))
```

Comment:
title and alt_title should be encoded as utf-8 cause it results in errors on special chars like in "marvel's iron fist" in my case - i also have some other things found and figure out what goes wrong

Reply:
Yeah, I've noticed that on other accounts. Unicode handling in 2.7 is a pain :(
Will you make changes to the PR or should I push an update?

Reply:
Push an update please

Reply:
Ignore it - i will push a pr to you if all is fixed - found another unicode bug :-( It sucks...